src/components/CldPlaceholder/CldPlaceholder.md

```diff
@@ -0,0 +1,46 @@
+### Attributes
+
+Refer to the base [Cloudinary JS SDK](https://github.com/cloudinary/cloudinary_js#configuration) for configuration options.
+
+See the [Image transformation reference](https://cloudinary.com/documentation/image_transformation_reference) documentation for all the options accepted by the `CldImage` and `CldTransformation` components.
+
+
+### Usage
+
+```jsx
+<cld-image cloudName="demo" publicId="small_dinosaur">
+  <cld-placeholder type="predominant-color"></cld-placeholder>
+</cld-image>
+```
+
+### Lazy mode
+
+Set the `lazy` property to only load the image when it needs to be displayed instead of when the page first loads.
```

Comment:
If that's a copy-paste from another place, it needs to be updated to use loading=lazy

Reply:
Updated

Reply:
The example was updated, but it still says that you need to set the 'lazy' priperty